README.md

```diff
@@ -11,30 +11,47 @@ extensions work, please see the following:
 
 ## Compilation / Usage
 
-To setup our extension, run the following (assuming Linux):
+First, you'll need to set up and build the [core Bareflank Hypervisor](https://github.com/bareflank/hypervisor),
+including [its SDK](https://github.com/bareflank/bfsdk).
 
-```
-cd ~/
-git clone https://github.com/Bareflank/hypervisor.git
-cd ~/hypervisor
+Once you have a working Bareflank build environment, building the extension is 
+easy:
+
+```sh
 git clone https://github.com/Bareflank/hypervisor_example_cpuidcount.git
 
-./tools/scripts/setup-<xxx>.sh --no-configure
-sudo reboot
+cd hypervisor_example_cpuidcount
+mkdir build
+cd build
+
+# Adjust the line below if you've installed to a non-default bfprefix location.
+cmake .. -D DCMAKE_TOOLCHAIN_FILE=~/bfprefix/cmake/CMakeToolchain_VMM.txt
```

Comment:
This should point to the prefix in the core hypervisor, not the ~/bfprefix. Those are just for unit testing so that we onlyt have to include in out Travis CI scripts what we need. Normal users should be using the prefixes that are created by the hypervisor super build.